circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
```

Comment:
Also, it seems that Circle CI version 1 is being used right now. Add `version: 2` at the top of this file to migrate to v2.0 [source](https://circleci.com/docs/2.0/sample-config/)

Reply:
Made the required changes in #289